src/lib/ui-switch/ui-switch.component.ts

```diff
@@ -30,11 +30,13 @@ const UI_SWITCH_CONTROL_VALUE_ACCESSOR: any = {
     [class.switch-medium]="size === 'medium'"
     [class.switch-small]="size === 'small'"
     [style.background-color]="getColor()"
-    [style.border-color]="getColor('borderColor')"
```

Comment:
why was this changed?

Reply:
It was a mistake, I fixed it in the last commit.